kyaml/fn/framework/log.go

```diff
@@ -0,0 +1,16 @@
+package framework
+
+import (
+	"fmt"
+	"os"
+)
+
+// Log prints to stderr.
+func Log(in ...interface{}) {
```

Comment:
To your own point about kyaml having too many responsibilities, I'm not sure we need to provide our own logging mechanism as part of the framework. There's nothing kyaml or k8s resource specific about this.